MC_Mix.cmd

```diff
@@ -32,12 +32,12 @@ action var tool done when ^Applying the final touches, you complete working
 action (work) off
 var alchemy.storage $MC_ALCHEMY.STORAGE
 
-action (order) var water.order $1 when (\d+)\)\..*10 splashes of water.*(Lirums|Kronars|Dokoras)
-action (order) var alcohol.order $1 when (\d+)\)\..*10 splashes of grain alcohol.*(Lirums|Kronars|Dokoras)
+# action (order) var water.order $1 when (\d+)\)\..*10 splashes of water.*(Lirums|Kronars|Dokoras)
+# action (order) var alcohol.order $1 when (\d+)\)\..*10 splashes of grain alcohol.*(Lirums|Kronars|Dokoras)
 action (order) var catalyst.order $1 when (\d+)\)\..*a massive coal nugget.*(Lirums|Kronars|Dokoras)
 
 var liquid tonic|wash|potion|elixir|draught
-var solid cream|salve|balm|poultices|ungent|ointment
+var solid cream|salve|balm|poultices|ungent|ointment|root

```

Comment:
Added root to solids, this way you can kick off mc_mix without mastercraft to create your own salves.